docs/resources/ces_alarmrule.md

```diff
@@ -58,6 +89,9 @@ The following arguments are supported:
   which indicates *critical*, *major*, *minor*, and *informational*, respectively.
   The default value is 2.
 
+* `alarm_type` - (Optional, String) Specifies the alarm type. The value can be **EVENT.SYS**, **EVENT.CUSTOM**
```

Comment:
better to note that which alarm_type is the default value.

Reply:
sure, fixed